src/AzureStack/CommerceAdmin/Commerce.Admin/generate.ps1

```diff
@@ -0,0 +1,11 @@
+# Remove old
+Remove-Item -Path "$PSScriptRoot\Generated" -Force -Recurse

```

Comment:
Why did you have to do this?

Reply:
I think I know why, like most other RPs/Language sdks you should really be setting the output-folder and other csharp settings in the configuration file [here](https://github.com/Azure/azure-rest-api-specs/blob/master/specification/commerce/resource-manager/readme.md)
This is not a blocking requirement so please remove this file and regenerate using the .cmd for now.
We highly recommend updating the configuration file though.

Reply:
Can I just leave it as it is for now until I update the spec to change the output folder.  Once I do that I can do another PR to update the generate.ps1.

Reply:
That works, please create an action item at your end to this effect.